modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -1,14 +1,49 @@
-[id="anchor-and-file-names"]
-= Anchor Names and File Names
+[id="module_anchor-and-file-names-concept"]
+= File Names and Anchors
 
-To optimize modular documentation, follow these guidelines for naming module anchors and files:
+To optimize modular documentation, follow these guidelines for naming files and creating anchors:
 
-Anchor names:: Provide an anchor in the format `+++[id="anchor-name_{context}"]+++` for every module so that it can be identified by Asciidoctor when reused or cross-referenced. `+++{context}+++` is a variable whose value you define in the assembly. Give the anchor the same or similar name as the module heading. Separate the words in the anchor with hyphens and use an underscore to separate the `+++{context}+++` part:
-+
+.File names
+Assembly and module file names should accurately and closely reflect the title of the assembly or module. File names should have the format `prefix-filename.adoc` or `prefix_filename.adoc` where `prefix` is one of the following module prefixes:
+
+* `con`: Concept module prefix
+* `proc`: Procedure module prefix
+* `ref`: Reference module prefix
+* `assembly`: Assembly module prefix
+
+.Examples
+* `con-guided-decision-tables.adoc`  (Concept module)
+* `proc-creating-guided-decision-tables.adoc`  (Procedure module for creating)
+* `proc-editing-guided-decision-tables.adoc`  (Procedure module for editing)
+* `ref-guided-decision-table-examples.adoc`  (Reference module with examples)
+* `ref-guided-decision-table-columns.adoc`  (Reference module with column types)
+* `assembly-designing-guided-decision-tables.adoc`  (Assembly of guided decision table modules)
+
+
+[NOTE]
+====
+Do not include special characters in file names. Ensure that all members of your team use the same file naming conventions.
+====
+
+These file naming guidelines are optional but highly recommended. However, if your team does not include the module prefixes in file names followed by either a hyphen (-) or an underscore (_), you must include one of the following variables in each concept, procedure, and reference module:
+
+NOTE: This is a Pantheon 2 requirement.
+
+[source]
+----
+:system-module-type: CONCEPT
+:system-module-type: PROCEDURE
+:system-module-type: REFERENCE
+----
+
+
+
+.Anchors
+At the top of every module, provide an anchor in the format `+++[id="filename_{context}"]+++` where `filename` is the exact name of the file, without the file extension (`.adoc`). Module anchors are necessary so that the module can be identified by Asciidoctor when reused or cross-referenced. The `{context}` variable is intentionally not defined in the module anchor ID. You define a context value in each assembly, in the format `:context: my-context-value`. This enables the context variable in each module to be populated according to the relevant assembly.
```

Comment:
Why does it have to be filename?

Reply:
This is not a hard requirement.

Reply:
@vikram-redhat Just the majority preference, and has a lot of conceptual and practical merit or many. One example is asciidoc processing tooling (rendering, validating, splitting, etc) is benefitted by this practice. This is definitely no reason to enforce it, but just like most FCC guidelines, mainly for those who just need to know a way to go and don't have strong reasons otherwise.

Reply:
Ah @sterobin that is fine and understandable. I am very well aware of the benefits this gives, as we use the same guideline in OpenShift docs (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata) and have built many tooling practices around that for docs.openshift.com.

This is the right thing to do for the benefits you mention. I think I have combined my concerns about long filenames which I mentioned in another part  and that then leading to really long module ids in this section. If we add/modify guideline to make the filenames not needing to be exactly as the title, I think we are good to ignore this part.